src/coreclr/debug/createdump/crashreportwriter.h

```diff
@@ -1,17 +1,41 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#pragma once
+
+#define JSON_INDENT_VALUE 1
+
 class CrashReportWriter
 {
 private:
+    int m_fd;
+    int m_indent;
+    bool m_comma;
     CrashInfo& m_crashInfo;
 
 public:
     CrashReportWriter(CrashInfo& crashInfo);
     virtual ~CrashReportWriter();
     void WriteCrashReport(const std::string& dumpFileName);
 
+    void WriteValue(const char* key, const char* value);
```

Comment:
If everything in this class is meant to be consumer only here, can we make these private?